plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -413,7 +411,7 @@ static void
 fu_plugin_thunderbolt_change (FuPlugin *plugin, GUdevDevice *device)
 {
 	FuDevice *dev;
-	const gchar *version;
+	g_autofree gchar *version;
```

Comment:
missing `= NULL`

Reply:
Oops. Not used to these g_auto* things yet.

I don't see these warnings when I run:
```
ninja -C build
```
Am I missing something?

Reply:
Looks like CI is showing them failing too.  CI runs with `-Werror`, see if they show up from that.